webapp/packages/core-blocks/src/Snackbars/useSnackbarTimeout.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * cloudbeaver - Cloud Database Manager
+ * Copyright (C) 2020 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import { useEffect } from 'react';
+
+interface IUseSnackbarTimeoutProps {
+  closeDelay?: number;
+  onClose?: (deletingDelay?: boolean) => void;
+  animate?: boolean;
+}
+
+export function useSnackbarTimeout({ closeDelay, onClose, animate = true }: IUseSnackbarTimeoutProps) {
```

Comment:
is this hook has any sense without closeDelay or onClose? i think better to declare args and seems no one of them can be optional